db/migrate/20161101175911_add_organization_listed_at_index.rb

```diff
@@ -0,0 +1,10 @@
+class AddOrganizationListedAtIndex < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    add_index :organizations,
+    :listed_at,
+    where: "(listed_at IS NOT NULL)",
+    algorithm: :concurrently
```

Comment:
The index can be made with a lock on the table as it's is a brand new table (no data). 

You should double check that both the scope where clauses satisfy this partial index clause otherwise one may be a table scan (probs not a big deal right now though). If it's not a partial index then you could also extract the index def to the table migration (i think).